sample_data/docker-compose-dl-streamer-example.yml

```diff
@@ -407,7 +413,11 @@ services:
     pids_limit: 1000
 
   mapping:
-    image: scenescape-mapping
+    profiles:
+      - experimental
+      - mapping
+    image: scenescape-mapping:${VERSION:-latest}
+    init: true
```

Comment:
out of curosity, why we need [init: true](https://docs.docker.com/reference/compose-file/services/#init) ?